### DISCUSSION TITLE
List of Supported Input Parameters in v3 from HF?

### DISCUSSION THREAD

**Discussion by souvikqb** [openai/whisper#1820] [Q&A]
Except Language, Tasks and Word Timestamps what other params are supported?

Asking this for Whisper v3 from HuggingFace - https://huggingface.co/openai/whisper-large-v3

**Comment by phineas-pta** [comment]
https://huggingface.co/docs/transformers/model_doc/whisper

**Reply by souvikqb** [reply]
Thanks for replying, but since Whisper v3 is from a transformers pipeline, I found these to be a little more relevant - 

1. ASR Pipeline - https://huggingface.co/docs/transformers/v4.35.2/en/main_classes/pipelines#transformers.AutomaticSpeechRecognitionPipeline

2. Text Generation - 
https://huggingface.co/docs/transformers/v4.35.2/en/main_classes/text_generation
